docs/api-reference/next/image.md

```diff
@@ -14,182 +14,222 @@ description: Enable Image Optimization with the built-in Image component.
 
 It should contain text that could replace the image [without changing the meaning of the page](https://html.spec.whatwg.org/multipage/images.html#general-guidelines). It is not meant to supplement the image and should not repeat information that is already provided in the captions above or below the image.
 
 If the image is [purely decorative](https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information) or [not intended for the user](https://html.spec.whatwg.org/multipage/images.html#an-image-not-intended-for-the-user), the `alt` property should be an empty string (`alt=""`).
 
-[Learn more](https://html.spec.whatwg.org/multipage/images.html#alt)
+Learn more about the [`alt` property](https://html.spec.whatwg.org/multipage/images.html#alt).
 
-## Optional Props
+### Optional Props
 
-The `<Image />` component accepts a number of additional properties beyond those which are required. This section describes the most commonly-used properties of the Image component. Find details about more rarely-used properties in the [Advanced Props](#advanced-props) section.
+The Image Component accepts a number of optional properties.
 
-### loader
+#### `loader`
 
 A custom function used to resolve image URLs.
 
-A `loader` is a function returning a URL string for the image, given the following parameters:
-
-- [`src`](#src)
-- [`width`](#width)
-- [`quality`](#quality)
+A `loader` is a function that returns a URL string for the image, it accepts the following parameters: `src`, `width`, `quality`.
 
-Here is an example of using a custom loader:
+```jsx:app/page.js
+import Image from 'next/image';
 
-```js
-import Image from 'next/image'
-
-const myLoader = ({ src, width, quality }) => {
-  return `https://example.com/${src}?w=${width}&q=${quality || 75}`
-}
+const imageLoader = ({ src, width, quality }) => {
+  return `https://example.com/${src}?w=${width}&q=${quality || 75}`;
+};
 
-const MyImage = (props) => {
+export default function Page({ imageLoader }) {
   return (
     <Image
-      loader={myLoader}
-      src="me.png"
+      loader={imageLoader}
+      src="/me.png"
       alt="Picture of the author"
       width={500}
       height={500}
     />
-  )
-}
+  );
+};
 ```
 
-Alternatively, you can use the [loaderFile](#loader-configuration) configuration in next.config.js to configure every instance of `next/image` in your application, without passing a prop.
+#### `fill`
 
-### fill
+```js
+fill={true} // {true} | {false}
+```
 
 A boolean that causes the image to fill the parent element instead of setting [`width`](#width) and [`height`](#height).
 
-The parent element _must_ assign `position: "relative"`, `position: "fixed"`, or `position: "absolute"` style.
-
 By default, the img element will automatically be assigned the `position: "absolute"` style.
+The parent element **must** assign `position: "relative"`, `position: "fixed"`, or `position: "absolute"` style.
```

Comment:
I liked this the way it was because the first sentence talks about the parent and the second sentence tells you about the parent and how to use it.

The rest is just default behavior which is optional.